packages/utils/test/parser/__snapshots__/schemaParser.test.ts.snap

```diff
@@ -1,5 +1,60 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`schemaParser() parse schema with array condition 1`] = `
+Object {
+  "-33d8d73c": Object {
```

Comment:
without this fix, this schema id `"-33d8d73c"` won't be generated for the condition and then causes errors when the form component tries to load this schema and looks for a validator with this id. See [codesandbox.io](https://codesandbox.io/s/kind-kate-d214qg?file=/src/ErrorApp.js) for error using this compiled schema